CHANGELOG.md

```diff
@@ -45,6 +45,7 @@
 - Minor: Add search to emote popup. (#3404)
 - Minor: Messages can now be highlighted by subscriber or founder badges. (#3445)
 - Minor: Add workaround for multipart emoji as described in [the RFC](https://mm2pl.github.io/emoji_rfc.pdf). (#3469)
+- Minor: Add context menu option to open the user's channel from a Usercard. (#3486)
```

Comment:
I don't think this is the best wording, but I feel like it's easier to understand for an average user.
```suggestion
- Minor: Added a way to open channel popup by right-clicking avatar in a usercard. (#3486)
```

Reply:
`the` avatar